app/features/receive/cashu-receive-quote.ts

```diff
@@ -31,6 +31,16 @@ export type CashuReceiveQuote = {
    * Date and time the receive quote expires in ISO 8601 format.
    */
   expiresAt: string;
+  /**
+   * Type of the receive.
+   * LIGHTNING - The money is received via Lightning.
+   * TOKEN - The money is received as cashu token. Those proofs are then used to mint tokens for the receiver's account via Lightning.
+   *         Used for cross-account cashu token receives where the receiver doesn't have the account for the mint of the received tokens so swap cannot be performed.
+   */
+  type: 'LIGHTNING' | 'TOKEN';
```

Comment:
I added this so we can also have a distinction on this entity if we did minting because of receiving actual lightning payment or because of doing cross account cashu token receive